tests/conftest.py

```diff
@@ -0,0 +1,526 @@
+import os
+import random
+import uuid
+from pathlib import Path
+
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.exc import ProgrammingError
+from sqlalchemy.orm.session import Session
+
+from app import create_app, test_db_url
+from config import Configuration
+from model import (create, Admin, Audience, Base, ExternalIntegration,
+                   Hyperlink, Library, Place, PlaceAlias,
+                   ServiceArea, get_one_or_create)
+from util import GeometryUtility
+
+TEST_DATA_DIR = Path(os.path.dirname(__file__)) / "data"
+
+
+@pytest.fixture(autouse=True, scope="session")
+def init_test_db():
+    """For a given testing session, pave and re-initialize the database"""
+    engine = create_engine(test_db_url)
+
+    for table in reversed(Base.metadata.sorted_tables):
+        try:
+            engine.execute(table.delete())
+        except ProgrammingError:
+            ...
+
+    with engine.connect() as conn:
+        Base.metadata.create_all(conn)
+
+    engine.dispose()
+
+
+@pytest.fixture(autouse=True, scope="session")
+def setup_test_adobe_integration(init_test_db):
+    """For a given testing session, set up an Adobe integration"""
+    engine = create_engine(test_db_url)
+    with engine.connect() as connection:
+        session = Session(connection)
+        (integration, _) = create(
+            session,
+            ExternalIntegration,
+            protocol=ExternalIntegration.ADOBE_VENDOR_ID,
+            goal=ExternalIntegration.DRM_GOAL
+        )
+        integration.setting(Configuration.ADOBE_VENDOR_ID).value = "VENDORID"
+        integration.setting(Configuration.ADOBE_VENDOR_ID_NODE_VALUE).value = "0x685b35c00f05"
```

Comment:
Just confirming this is a made-up value.

Reply:
So far as I know, though I didn't make it up myself--I just ported it directly into a fixture from what's in here: https://github.com/NYPL-Simplified/library_registry/blob/548fc9f73a3c871ad2a618ae0eab3bac38411620/tests/test_adobe_vendor_id.py#L34